context/app/static/js/pages/Organ/Organ.tsx

```diff
@@ -68,15 +85,15 @@ function Organ({ organ }: OrganProps) {
             <Azimuth config={organ.azimuth} />
           </Section>
         )}
-        {shouldDisplaySection[assaysId] && (
+        {shouldDisplaySection[assaysId] && searchHasData && (
```

Comment:
Could `searchHasData` be included as part of the `shouldDisplaySection` logic above?